shared/build.gradle.kts

```diff
@@ -38,6 +38,7 @@ kotlin {
     sourceSets {
         val ktorVersion = "2.3.3"
         val loggingVersion = "1.3.0"
+        val voyagerVersion = "1.0.0-rc05"
```

Comment:
not sure u wanna try the 1.0.0-rc06
all voyager screen will support Android lifecycle properly, don’t required AndroidScreen anymore 🤔

Reply:
Ok, I just copied the version from the documentation (still 1.0.0-rc05), I can try the new version, not that we are in a production system and we have to play safe :)